TASVideos/Pages/Shared/Components/FrontpageSubmissionList/Default.cshtml

```diff
@@ -14,7 +14,7 @@
 				<td><timezone-convert asp-for="@item.Submitted" date-only="true" /></td>
 				<td>@item.System @item.GameName @(!string.IsNullOrWhiteSpace(item.Branch) ? $"\"{item.Branch}\"" : "")</td>
 				<td><sub-link id="item.Id">@(item.Time.ToStringWithOptionalDaysAndHours())</sub-link></td>
-				<td>@item.Author</td>
+				<td>@(string.Join(", ",(item.Authors ?? Enumerable.Empty<string>()).Concat((item.AdditionalAuthors ?? "").SplitWithEmpty(","))).LastCommaToAmpersand())</td>
```

Comment:
All of this logic is complicated and repeated, I'd like to see it in a method somewhere